src/Wrapper.js

```diff
@@ -354,8 +354,10 @@ class Wrapper extends React.PureComponent {
           daoAddress={daoAddress}
           params={params}
           repos={repos}
+          canUpgradeOrg={this.props.canUpgradeOrg}
```

Comment:
If you can get this via props maybe you don't need to pass it as an arg to `renderApp`?